### ISSUE TITLE
invalid syntax on upcoming 1.7

### ISSUE THREAD

**Issue by KristofferC** [JuliaGPU/KernelAbstractions.jl#260]
Comes from the macroexpansion in `@ka_code_typed`AFAIU:

```
julia> include("test/compiler.jl")
ERROR: LoadError: syntax: invalid syntax (escape 1)
Stacktrace:
 [1] top-level scope
   @ ~/Documents/JuliaPkgs/KernelAbstractions.jl/test/compiler.jl:40
 [2] include(fname::String)
   @ Base.MainInclude ./client.jl:451
 [3] top-level scope
   @ REPL[1]:1
in expression starting at /Users/kristoffercarlsson/Documents/JuliaPkgs/KernelAbstractions.jl/test/compiler.jl:40
```

**Comment by KristofferC** [comment]
Nvm, was on wrong branch.

**Comment by vchuravy** [comment]
Yeah fixed on master/0.7
